File: src/main/java/org/jukeboxmc/potion/PoisonEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class PoisonEffect extends Effect {

    @Override
    public int getId() {
        return 19;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.POISON;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 78, 147, 49 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/PoisonEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class PoisonEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 19
    }

    override fun getEffectType(): EffectType {
        return EffectType.POISON
    }

    override fun getEffectColor(): Color {
        return Color(78, 147, 49)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}